engine/src/main/java/io/zeebe/engine/processing/streamprocessor/MetadataEventFilter.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.engine.processing.streamprocessor;
+
+import io.zeebe.logstreams.log.LoggedEvent;
+import io.zeebe.protocol.impl.record.RecordMetadata;
+
+public class MetadataEventFilter implements EventFilter {
```

Comment:
I agree. I created a new issue to align the names (#6245). There are a bunch of places to rename. Since it is not required to continue with the topic, I would prefer to do it after the major parts of the migration are done.